code/datums/wires/explosive.dm

```diff
@@ -11,6 +11,45 @@
 /datum/wires/explosive/proc/explode()
 	return
 
+/datum/wires/explosive/chem_grenade
+	holder_type = /obj/item/grenade/chem_grenade
+	randomize = TRUE
+	var/fingerprint
+	var/assembly
+
+/datum/wires/explosive/chem_grenade/interactable(mob/user)
+	var/obj/item/grenade/chem_grenade/G = holder
+	if(G.stage == 2)

```

Comment:
not defined?

Reply:
defining wont work

Reply:
the 2 isn't defined anywhere...?

Reply:
its defined and  undefined in chem_grenade

Reply:
move the define to an appropriate location in _DEFINES and don't undefine it, it should be used here.